### PR TITLE
Fix hyperparameter naming in readme, add learning resources

### PR DESCRIPTION
**Pull request by rafajak**
Hello there, 

I made some changes in the hyperparameters section to reflect the argument names the model accepts, namely:

- layers -> num_layers
- dropout -> output_keep_prob

I also changed the suggested value of the latter from 0.25 to 0.75, as I'm assuming the author suggests dropout rate of 0.25, which corresponds to output_keep_prob of 0.75 in TensorFlow's DropoutWrapper. 

Additionally, I added the learning resources Daniel lists in issue #7.

Thanks so much for your work on ml5 and tutorials, it's an absolute joy to play with. :) 

Cheers,
Raf

### CODE REVIEW

**Comment by shiffman** [comment]
Thank you for this, sorry for the long delay is taking a look!
